DESCRIPTION

```diff
@@ -21,10 +21,7 @@ Imports:
 Suggests:
     highr,
     Cairo,
-    testthat,
-    data.table,
-    dplyr,
-    htmlwidgets
+    testthat
 Enhances: data.table, dplyr, htmlwidgets
```

Comment:
my point is that dplyr belongs into `Suggests` (we use it to run tests but repr doesn’t provide methods for its classes).

other than that everything is awesome